tests/test_carsons.py

```diff
@@ -136,6 +137,21 @@ def expected_z_abc_three_neutrals():
                 [-14 + 0j, -14 + 0j, -13 + 0j]])
 
 
+def z_abc_kersting_4_3():
+    return array([
+                [0.4576+1.078j, 0.1560+0.5017j, 0.1535+0.3849j],
```

Comment:
I think we already have these calculations written out for the 4.3 test case from Kerstings, since we are doing the geometric -> phase-impedance calculation. what do you think about putting these tests in the same file? and using the same unit conversion here https://github.com/opusonesolutions/carsons/blob/1b1fda652d17a5119297bee4029a6c9f1373f88c/tests/test_concentric_neutral_cable.py#L72 so that the test inputs / outputs can be mapped back to the reference material?

you'll see in that file that we've also inputted all the geometric line characteristics in the imperial units that the book uses, and converted them programmatically to metric / SI -- it makes it much easier to verify against the source material

Reply:
A good idea, though I just noticed it was example 4.1 (from the 3rd edition), don't think that's already in the test cases

Reply:
ok yes, i have the same numbers in 4.1. ah, and i suppose since we aren't actually doing any per-length conversions in this computation, the units just pass through. so, it doesn't really matter too much in that case. but, it might be a good idea to rename or add a docstring indicating that you got this from example 4.1 in kersting (3e). just makes it easy to track